virtio-devices/src/transport/pci_device.rs

```diff
@@ -1179,7 +1179,14 @@ impl PciDevice for VirtioPciDevice {
             o if (NOTIFICATION_BAR_OFFSET..NOTIFICATION_BAR_OFFSET + NOTIFICATION_SIZE)
                 .contains(&o) =>
             {
+                #[cfg(feature = "sev_snp")]
+                for (_event, _addr) in self.ioeventfds(_base) {
+                    if _addr == _base + offset {
+                        _event.write(1).unwrap();
```

Comment:
Since we don't register ioeventfd in case of SEV-SNP guests. So why do we need to write these events to ioeventfd?

Reply:
Commit ID: d0be450b1b

Reply:
Thats the reason it writes to the bar as a notifications. Orthr wise it would have been handled by ioeventfds.